tests/platform_tests/mellanox/test_check_sfp_using_ethtool.py

```diff
@@ -21,17 +22,26 @@ def test_check_sfp_using_ethtool(duthost, conn_graph_facts):
     ports_config = json.loads(duthost.command("sudo sonic-cfggen -d --var-json PORT")["stdout"])
 
     logging.info("Use the ethtool to check SFP information")
+    if conn_graph_facts["device_info"]["HwSku"] in SPC3_HWSKUS:
```

Comment:
It is better to use `duthost.facts["hwsku"]` here.

Reply:
Fixed.